app/src/client/components/buildPage/UserProperty.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState, memo } from 'react';
+
+import _ from 'lodash';
+import { getModels, useQuery } from 'wasp/client/operations';
+
+import CustomBreadcrumb from '../CustomBreadcrumb';
+import Button from '../Button';
+import ModelsList from '../ModelsList';
```

Comment:
Just a general comment not related to this PR: maybe we could consider to use "@/" style imports.

Reply:
Sure, in that case lets make "@/" style imports as a standard way to import modules in our project. I will raise a new PR with this change.